Sources/SwiftGen/Commander/ConfigCommands.swift

```diff
@@ -162,3 +160,20 @@ enum ConfigCLI {
     NSWorkspace.shared.open(docURL)
   }
 }
+
+private extension Config {
+  func runCommands(verbose: Bool) throws {
+    let errors = commands.keys.sorted().parallelCompactMap { cmd -> Swift.Error? in
+      do {
+        _ = try run(cmd: cmd, verbose: verbose)
+        return nil
+      } catch {
+        return error
+      }
+    }
+
+    if let error = errors.first {
```

Comment:
we could create an error type to combine multiple errors into one, put i thought this was a good enough solution for now (since in the existing implementation the user will only get the first error that they've faced)

Reply:
Lol, I read your code from Fork.app (as when I did a `git fetch` I saw that new branch and was curious) before seeing that draft PR, and thought exactly of this. Great minds ;)

I think it would be easy enough though to add an `indirect case composite([Config.Error])` to the `enum Config.Error`, with its `description`  being:
 `case .composite(let errors): return errors.map({ " - \($0.description)" }).joined(separator: "\n")` for example.